zdm/parameters.py

```diff
@@ -146,18 +146,21 @@ class RepeatParams(data_class.myDataClass):
 @dataclass
 class MWParams(data_class.myDataClass):
     ISM: float = field(
-        default=35.0,
-        metadata={"help": "Assumed DM for the Galactic ISM in units of pc/cm^3"},
-    )
+        default=35.,
+        metadata={'help': 'Assumed DM for the Galactic ISM',
+                  'unit': 'pc cm$^{-3}$',
+        })
+    uDMG: float = field(
```

Comment:
How annoying would it be to replace "u" with "sigma"? The reason is that "u" is awfully like "\mu", which is a mean. If this only appears O~10 times in the code, it's be great to rename this to sigmaDMG... (we also use "sigma" for host galaxy uncertainty)